build.ps1

```diff
@@ -118,6 +118,8 @@ $PACKAGES_CONFIG_MD5 = Join-Path $TOOLS_DIR "packages.config.md5sum"
 $ADDINS_PACKAGES_CONFIG = Join-Path $ADDINS_DIR "packages.config"
 $MODULES_PACKAGES_CONFIG = Join-Path $MODULES_DIR "packages.config"
 
+$env:CAKE_PATHS_TOOLS = $TOOLS_DIR
```

Comment:
We should also set `CAKE_PATHS_ADDINS` and `CAKE_PATHS_MODULES` to the appropriate values